web/client/api/geoserver/Importer.js

```diff
@@ -55,9 +55,15 @@ var Api = {
         let url = geoserverBaseUrl + "imports/" + importId + "/tasks/" + taskId + "/target";
         return axios.get(url, options);
     },
-    runImport: function( geoserverBaseUrl, importId, options) {
+    runImport: function( geoserverBaseUrl, importId, options = {}) {
         let url = geoserverBaseUrl + "imports/" + importId + "?async=true";
-        return axios.post(url, null, options);
+        return axios.post(url, null, {
+            ...options,
```

Comment:
I think you are duplicating headers, this way. I would extract headers from options

Reply:
yes, good proposal, it could be more readable.